src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -13738,6 +13815,14 @@ int Assembler::simd_prefix_and_encode(XMMRegister dst, XMMRegister nds, XMMRegis
   }
 }
 
+bool Assembler::is_demotable(bool no_flags, int dst_enc, int nds_enc, int src_enc) {
```

Comment:
src_enc is not being used in this method so could be removed.

Reply:
Please see the src_enc removed in the updated code.